maven-plugin-plugin/src/it/v4api/invoker.properties

```diff
@@ -15,4 +15,5 @@
 # specific language governing permissions and limitations
 # under the License.
 
+invoker.java.version = 17+
```

Comment:
Confusing if the entire project require Java 17, why here especially?

Reply:
The entire project is JDK 8.  The mojos run on Maven 3.x + JDK 8 if you want to build Maven 3.x plugins.  If you build a Maven 4.x plugin, you'll need JDK 17.  That's the case for this IT.  Other ITs are unchanged.

The migration to Maven 4 only should imho happen at the same time as m-plugin-p is migrated to be a Maven 4 plugin.